test/Sleep-Repository-test.js

```diff
@@ -1,20 +1,19 @@
 import {expect} from "chai";
 import SleepRepository from "../src/SleepRepository";
-import Sleep from "../src/Sleep";
 import UserRepository from '../src/UserRepository';
 
-describe('SleepRepository', function() {
+describe("SleepRepository", function () {
   let sleep1, sleep2, sleep3, mockRawData, mockSleepRepo, todaysDate;
   beforeEach(() => {
-    sleep1 = {
+    sleep1 =  {
       "userID": 1,
-      "date": "2019/06/15",
+      "date": "2019/06/17",
```

Comment:
What was the reasoning for changing the year?

Reply:
I was checking to see it's behavior in a sad path